awesomebook.css

```diff
@@ -0,0 +1,137 @@
+/* stylelint-disable indentation */
+/* stylelint-disable csstree/validator */
+/* stylelint-disable selector-class-pattern */
+body {
+  text-align: center;
+}
+
+.main-container {
+  max-width: 500px;
```

Comment:
- [ ] Kindy check your page width it should fit the window so as to fully match the wireframe layout.

![image](https://user-images.githubusercontent.com/61408860/192796173-e5f5e33f-4a47-44b7-925f-b63c1f7cade3.png)

Reply:
We removed the width so that it fits the whole wireframe.